src/tasks/telegram_worker.js

```diff
@@ -26,6 +26,38 @@ async function fetchEvents(date) {
         })
 }
 
+function filterEvents(event) {
+    //no offense but these groups aren't tech related
+    return (event.group_name !== "Kakis SG Anything Watever Meetup Group") &&
```

Comment:
I believe there should be an upstream solution. We blacklist groups. But probably need to fix that at the crawler filter eventually.

Reply:
I personally have coded extra (more strict and extensive?) check on my side too. Maybe could help a little 🙏  https://github.com/cheeaun/slakoth/blob/ba35b66cc49d928b29a34b39cf6342b40c540ddf/server.js#L44-L97